### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thatnecromancerlife**
@littlelark Please review this readme before I commit it to make sure I haven't forgotten any information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
-# yuggoth
+#Yuggoth
 A collection of blank maps, sorted by genre then type.
+
+##Purpose
+The purpose of this repo is to provide maps for roleplay settings.
```
Comment (littlelark) [inline README.md]: We could probably condense these first two sections into one. (It's pretty standard that the first line under the project name is the project's purpose from what I've seen.)

```suggestion
# Yuggoth
A collection of blank maps for roleplay settings, sorted by genre, then type.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+
+##Commission Agreement
+For all commissioned maps, two copies will be made: the requester's version, including names, and our version, without names. Like our other resources, at the time tha t a site comes to a close, any maps that were made will be added to our collection and made available under a Creative Commons Zero license.
```
Comment (littlelark) [inline README.md:8]: Caught an extra space trying to sneak in!

```suggestion
For all commissioned maps, two copies will be made: the requester's version, including names, and our version, without names. Like our other resources, at the time that a site comes to a close, any maps that were made will be added to our collection and made available under a Creative Commons Zero license.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,13 @@
-# yuggoth
+#Yuggoth
 A collection of blank maps, sorted by genre then type.
+
+##Purpose
+The purpose of this repo is to provide maps for roleplay settings.
+
+##Commission Agreement
```
Comment (littlelark) [inline README.md]: If I remember right, GitHub needs a space between the pound signs and the next word in order to display headings correctly.

```suggestion
## Commission Agreement
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,13 @@
...
+For all commissioned maps, two copies will be made: the requester's version, including names, and our version, without names. Like our other resources, at the time tha t a site comes to a close, any maps that were made will be added to our collection and made available under a Creative Commons Zero license.
+
+##Disclaimers
```
Comment (littlelark) [inline README.md]:
```suggestion
## Disclaimers
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,13 @@
...
+While all maps found within this repo are under the CC0 license, please keep in mind the following:
+
+A Creative Commons Zero license does not waive liability for use of copyrighted material. While these maps are open and available for use, modification, and distribution, both Inkarnate and Campaign Cartographer 3 retain copyrights to their artwork, icons, etc.  These individual images may not be distributed unless part of a larger work, i.e. a map.
```
Comment (littlelark) [inline README.md:13]: This bit is worded really well, love it. <3

**Review by littlelark — COMMENTED** [review]
Made some notes for some minor tweaks, but overall this is looking good!

**Review by thatnecromancerlife — COMMENTED** [review]
Ready for approval. Thank you!

**Pull request merged**